src/app_requirements.txt

```diff
@@ -22,7 +22,7 @@ newrelic==4.20.1.121
 openapi-spec-validator==0.2.7
 pathlib==1.0.1
 psutil==5.6.3
-pynamodb==3.4.1
```

Comment:
We were required to upgrade this to support the new queries being used.

Reply:
Is there a question here?

Reply:
Nope.  I was merely calling out this for other readers.